src/metrics/chainflip/gaugePendingRedemptions.ts

```diff
@@ -0,0 +1,26 @@
+import promClient, {Gauge} from "prom-client";
+import {Context} from "../../lib/interfaces";
+
+const metricName: string = "cf_pending_redemptions"
+const metric: Gauge = new promClient.Gauge({
+    name: metricName,
+    help: "Pending redemptions registered on the state-chain.",
+    registers: [],
+});
+
+export const gaugePendingRedemptions = async (context: Context): Promise<void> => {
+    const {logger, api, registry, metricFailure} = context;
+
+    if (registry.getSingleMetric(metricName) === undefined)
+        registry.registerMetric(metric);
+
+    metricFailure.labels({metric: metricName}).set(0);
+
+    logger.debug(`Scraping ${metricName}`);
+    try {
+        const pendingRedemptions = await api.query.funding.pendingRedemptions.entries();
+        metric.set(pendingRedemptions.length);
```

Comment:
Didn't we want to track if there was big redemptions? I feel here it is the perfect place to do so since we have access to all the redemptions

Reply:
Will do in another PR. Bit more complicated to calculate than I thought.